atlp-rwanda/rca-phantom-team2-bn
# Ft simulate bus motion

### What does this PR do?

Adds the socket functionality for changing speed
### Description of Task to be Completed?
The driver should be able to change the speed: slow down or speed up

### How should this be manually tested?
Clone the repo
Run `npm install` to install modules
Run `npm run dev` to run the application and **make sure you have the client and driver side**(the 2 socket part)
